### DISCUSSION TITLE
[SOLVED] Internal player for podcasts

### DISCUSSION THREAD

**Discussion by chromer030** [martinrotter/rssguard#1216] [Ideas]
RSSGuard have a full potential to be a podcast player too by adding a player. (I'm aware of adding external player command).
What is your opinion about this?

**Comment by martinrotter** [comment]
Hi, working on this.

**Reply by chromer030** [reply]
Thanks for consideration, i creating a FR for further tracking related issues. : https://github.com/martinrotter/rssguard/issues/1226
